### PR TITLE
show amp ad instructions on demo

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

File: src/60_AMP_Ads/AMP_Ads_vs_non-AMP_Ads.html [hunk 1]
```diff
@@ -22,6 +22,8 @@
 </head>
 <body>
 
+  <p class="p4">This sample compares the performance of an AMP Ad vs the a regular ad. To best compare these two, either view this page on a mobile device with active 3G connection or, if you are on Desktop, enable <a href="https://developers.google.com/web/tools/chrome-devtools/network-performance/network-conditions">network throttling</a> in Chrome Dev Tools (e.g. "Regular 3G").</p>
```
Comment (kul3r4) [inline src/60_AMP_Ads/AMP_Ads_vs_non-AMP_Ads.html]: vs the a regular ad -> vs a regular ad

Comment (sebastianbenz) [inline src/60_AMP_Ads/AMP_Ads_vs_non-AMP_Ads.html]: done

**Comment by sebastianbenz** [comment]
PTAL

**Pull request merged**